homeassistant/components/p1_monitor/sensor.py

```diff
@@ -250,15 +252,16 @@ def __init__(
         entry_id: str,
         description: SensorEntityDescription,
         service_key: str,
+        name: str,
         service: str,
     ) -> None:
         """Initialize P1 Monitor sensor."""
         super().__init__(coordinator=coordinator)
         self._service_key = service_key
 
-        self.entity_id = f"{SENSOR_DOMAIN}.{service_key}_{description.key}"
+        self.entity_id = f"{SENSOR_DOMAIN}.{name}_{description.key}"
         self.entity_description = description
-        self._attr_unique_id = f"{entry_id}_{service_key}_{description.key}"
+        self._attr_unique_id = f"{entry_id}_{name}_{description.key}"
```

Comment:
The entry title should not be part of the unique id since the title is configurable.

Reply:
Alright, revert that part back